src/app/core/services/issue.service.ts

```diff
@@ -87,15 +87,14 @@ export class IssueService {
           return EMPTY;
         }
         return this.githubService.fetchIssueGraphql(issueId).pipe(
-            map((response) => {
-              const issue = this.createIssueModel(response);
-              this.updateLocalStore(issue);
-              return issue;
-            }),
-            catchError((err) => this.getIssue(issueId))
-          );
-        }
-      )
+          map((response) => {
+            const issue = this.createIssueModel(response);
+            this.updateLocalStore(issue);
+            return issue;
+          }),
+          catchError((err) => this.getIssue(issueId))
+        );
+      })
```

Comment:
There is no change to this part of the file, a formatting change is not relevant to this PR. Let's remove this change from the PR.

if your ide is automatically formatting this, there should be an option to make it only format parts you modify.